test/TestSetup.sol

```diff
@@ -411,6 +407,11 @@ contract TestSetup is Test {
         bucketRateLimiter.setCapacity(40 ether);
         bucketRateLimiter.setRefillRatePerSecond(1 ether);
 
+        if (forkEnum == MAINNET_FORK || forkEnum == TESTNET_FORK) {
```

Comment:
what's the reason to limit this. Wouldn't we always want to upgrade to latest version, even if on say a tenderly fork?

Reply:
I was also unsure of the purpose, but I was just keep this existing logic intact:
https://github.com/etherfi-protocol/smart-contracts/blob/2cebfd7e87cca237d4444b4632d22aff439ce132/test/TestSetup.sol#L398-L404

Noted as something to look into for the test refactor